lib/membrane_rtmp_plugin/rtmp/source/message_handler.ex

```diff
@@ -244,6 +247,16 @@ defmodule Membrane.RTMP.MessageHandler do
     :gen_tcp.send(socket, [header | payload])
   end
 
+  defp get_validation_action(state, result) do
+    notification =
+      case result do
+        {:ok, msg} -> {:notify, {:stream_validation_success, msg}}
+        {:error, msg} -> {:notify, {:stream_validation_error, msg}}
```

Comment:
`reason` instead of `msg` would be more readable.

Reply:
And how about naming it `validation_action` instead of `get`? You are in fact not accessing but creating it so the `get` looks quite weird.

Reply:
okay, I just prefer to have some verb in the function name